### ISSUE TITLE
Change engine to model

### ISSUE THREAD

**Issue by takenek** [knrd1/chatgpt#8]
https://platform.openai.com/docs/api-reference/engines

The Engines endpoints are deprecated.
Please use their replacement, [Models], instead.

If i good understand we need change Engine to model because it's deprecated.

**Comment by knrd1** [comment]
Good idea, thank you. This has been done. The future plan is to move from v1/completions endpoint to v1/chat/completions endpoint. This requires to use openai.ChatCompletion.create with different parameters. But it will allow us to use model gpt 3.5 turbo which is 10 times cheaper than davinci 003.
